Orange/widgets/unsupervised/tests/test_owcorrespondence.py

```diff
@@ -37,3 +37,18 @@ def test_data_values_in_column(self):
                 "klkk"
             )))
         self.send_signal("Data", table)
+
+    def test_data_one_value_zero(self):
+        """
+        Check that the widget does not crash on discrete attributes with only
+        one value.
+        GH-2149
+        """
+        table = Table(
+            Domain(
+                [DiscreteVariable("a", values=["0"])]
+            ),
+            list(zip(
+                ["0", "0", "0"]
+            )))
+        self.send_signal("Data", table)
```

Comment:
Cool trick. You will get the same results with `list(zip("000"))`.

However, `[('0',)]*3` or simply `[(0,), (0,), (0,)]` or `np.zeros((3, 1))` would be more reader-friendly.